docs/db-migration/prototypes/admin-settings.html

```
@@ -0,0 +1,523 @@
+                                <th class="h-9 px-4 text-left text-xs font-medium uppercase w-48">Field</th>
+                                <th class="h-9 px-4 text-left text-xs font-medium uppercase w-36">Visibility</th>
+                                <th class="h-9 px-4 text-left text-xs font-medium uppercase w-24">Required</th>
+                                <th class="h-9 px-4 text-left text-xs font-medium uppercase">Default Value</th>
+                                <th class="h-9 px-4 text-left text-xs font-medium uppercase">Description</th>
+                            </tr>
+                        </thead>
+                        <tbody class="text-sm">
+                            <tr>
+                                <td class="h-11 px-4 font-mono text-xs font-medium">name</td>
+                                <td class="h-11 px-4"><span class="vis-editable text-xs font-medium">editable</span> <span class="text-xs text-muted">(locked)</span></td>
+                                <td class="h-11 px-4"><input type="checkbox" checked disabled></td>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)